src/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from "react";
+import { CONTINUE_TO_SHOPPING } from "../../enums/not-found";
+import IDiv from "../../interfaces/div";
+import "../../styles/components/_not-found.scss";
+import { variantTypes } from "../../types/button";
+import { colorTypes } from "../../types/color";
+import { sizeTypes } from "../../types/icon";
+import { Box } from "../box";
+import Button from "../button";
+import Icon from "../icon";
+import Typography from "../typography";
+
+export default class NotFound extends PureComponent<INotFoundProps> {
+  public render() {
+    const {
+      title,
+      subtitle,
+      buttonText = CONTINUE_TO_SHOPPING,
+      buttonVariant = "secondary",
+      buttonOnClick,
+      icon = "not-found",
+      iconSize = "xlarge",
+      iconColor = "dark-gray",
+      stroke = 0,
+    } = this.props;
+
+    return (
+      <Box className="q-not-found" spaced textAlign="center">
+        <Icon className="q-icon"
```

Comment:
q-icon class refers to Icon component itself. It would be better if the name of this class is changed to &__icon